projects/core/src/state/utils/entity-loader/entity-loader.action.ts

```diff
@@ -26,9 +27,11 @@ export interface EntityLoaderAction extends Action {
   readonly meta?: EntityLoaderMeta;
 }
 
+export type EntityLoadId = string | string[] | null;
```

Comment:
Is this change in the scope of the task? If not, let's do it later in a separate PR.

Reply:
Since I touched this file, Sonar Cloud threw me an error, hence the change.

Reply:
In that case, I propose to change the name to `EntityId`

Reply:
As we discussed, there are more changes related to this union type, so let's move this to a separate ticket.